Backends/CLX/frame-manager.lisp

```diff
@@ -126,17 +89,13 @@
                  ,concrete-pane-class-symbol)
               ()
               (:metaclass ,(type-of (find-class concrete-pane-class-symbol))))))
-        ;; (format *debug-io* "dummy class mirror ~A: ~A~%" concrete-pane-class-symbol class-symbol)
         (setf concrete-pane-class (find-class class-symbol)))))
   concrete-pane-class)
 
 (defmethod make-pane-1 ((fm clx-frame-manager) (frame application-frame) type &rest args)
-  (apply #'make-instance
-	 (maybe-mirroring fm (find-concrete-pane-class type))
-	 :frame frame
-	 :manager fm
-	 :port (port frame)
-	 args))
+  (apply #'make-instance (find-concrete-pane-class fm type)
+	 :frame frame :manager fm :port (port frame)
+         args))
```

Comment:
Multiple backends have this exact method. Could this become a default method?

Reply:
good idea.

Reply:
fixed